model/riscv_sys_control.sail

```diff
@@ -145,6 +147,18 @@ function getPendingSet(priv : Privilege) -> option((xlenbits, Privilege)) = {
   else None()
 }
 
+/* Check if a locally enabled interrupt is pending.
+ *
+ * This does not examine the global enable bits (MIE and SIE in
+ * mstatus) and their delegation in mideleg. It does honor the
+ * local interrupt enables in mie.
+ *
+ * This is used when the hart is in the Waiting state.
+ */
```

Comment:
The documentation might benefit from pointing out that this is specifically used by the WFI instruction (and any of its kindred, perhaps) and citing and quoting [the relevant prose from the privileged spec](https://five-embeddev.com/riscv-priv-isa-manual/Priv-v1.12/machine.html#wfi):
> The operation of WFI must be unaffected by the global interrupt bits in  mstatus (MIE and SIE) and the delegation register mideleg (i.e., the hart must resume if a locally enabled interrupt becomes pending, even if it has been delegated to a less-privileged mode), but should honor the individual interrupt enables (e.g, MTIE) (i.e., implementations should avoid resuming the hart if the interrupt is pending but not individually enabled).

Reply:
Ok.